example/svrg_module/README.md

```diff
@@ -0,0 +1,26 @@
+## SVRG Module Example
+
```

Comment:
Introduce what it is as well. From your other document:
"SVRGModule is an extension to the Module API that implements SVRG (Stochastic Variance Reduced Gradients) optimization ..."